apps/admin-gui/src/app/shared/components/add-owner-dialog/add-owner-dialog.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+import { NotificatorService } from '@perun-web-apps/perun/services';
+import { OwnersManagerService } from '@perun-web-apps/perun/openapi';
+import { TranslateService } from '@ngx-translate/core';
+import { FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'perun-web-apps-add-owner-dialog',
+  templateUrl: './add-owner-dialog.component.html',
+  styleUrls: ['./add-owner-dialog.component.scss']
+})
+export class AddOwnerDialogComponent implements OnInit {
+
+  constructor(
+    private dialogRef: MatDialogRef<AddOwnerDialogComponent>,
+    private notificator: NotificatorService,
+    private ownersManagerService:OwnersManagerService,
+    private translate: TranslateService
+  ) {
+    translate.get('DIALOGS.ADD_OWNER.SUCCESS').subscribe(value => this.successMessage = value);
+  }
+
+  successMessage: string;
+  loading: boolean;
+
+  nameCtrl: FormControl;
+  contactCtrl: FormControl;
+  type = '1';
+
+  ngOnInit() {
+    this.nameCtrl = new FormControl(null, [Validators.required, Validators.pattern('^[\\w.-]+( [\\w.-]+)*$')]);
+    this.contactCtrl = new FormControl(null, [Validators.required, Validators.pattern('^[\\w.-]+( [\\w.-]+)*$')]);
+  }
+
+  onCancel() {
+    this.dialogRef.close(false);
+  }
+
+  onSubmit() {
+    this.loading = true;
+    // @ts-ignore
+    this.ownersManagerService.createOwner({name: this.nameCtrl.value, contact: this.contactCtrl.value, ownerType: Number.parseInt(this.type)}).subscribe(() => {
```

Comment:
My IDE throws the TSLint error for the `Number.parseInt(this.type)`. You can try `Number(this.type)` or `Number.parseInt(this.type, 10)`.